src/Search/Workflow/AbstractWorkflow.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace eLife\Search\Workflow;
+
+abstract class AbstractWorkflow
+{
+    abstract public function getSdkClass();
+
+    abstract public function index($entity);
+
+    abstract public function insert(string $json, string $id, bool $skipInsert);
+
+    abstract public function postValidate(string $id, bool $skipValidate);
+
+    public function run($entity) {
+        $skipInsert = false;
+        $skipValidate = false;
```

Comment:
Are `skipInsert` and `skipValidate` ever true?

Reply:
Yes, they are true in ReviewedPreprintWorkflow